src/backend/src/controllers/projects.controllers.ts

```diff
@@ -203,6 +203,16 @@ export default class ProjectsController {
     }
   }
 
+  static async getAllMaterialTypes(req: Request, res: Response, next: NextFunction) {
+    try {
+      const user = await getCurrentUser(res);
+      const materialTypes: Material_Type[] = await ProjectsService.getAllMaterialTypes(user);
```

Comment:
this should be a shared type that u add, and you should update the transformer to return that shared type (this is bc we can't access the prisma types in the frontend). I'm realizing the getAllManufacturers does the same thing, would u be able to update that one as well